markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -141,7 +141,8 @@ library PerpsAccount {
             );
         }
 
-        availableWithdrawableCollateralUsd = availableMargin - requiredMaintenanceMargin;
+        // availableMargin can be assumed to be positive since we check for isEligible for liquidation prior
+        availableWithdrawableCollateralUsd = availableMargin.toUint() - requiredMaintenanceMargin;
```

Comment:
This is fixed in my PR, but this statement needs to be above the if statement on line 137

Reply:
oh yeah; we should get that merged.  i'll fix it here so maybe you can rebase